tests/terraform/create_image.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create HDD image that boots automatically and add a second
```

Comment:
you not creating something but modifying existing one , right ?

Reply:
hm , maybe module also needs to be renamed ?

Reply:
I am modifying the one that comes from create_hdd_textmode. What name do you suggest?

Reply:
modify_image.pm